packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -224,6 +224,43 @@ class ServiceExtensionSelector {
   }
 }
 
+class SamplePeriodSelector {
```

Comment:
Nit: this class seems a bit specific to be in ui_utils. Maybe move it to lib/src/ui/vm_flags.dart
or something.